go/kbtest/chat.go

```diff
@@ -246,11 +250,17 @@ func (m *ChatRemoteMock) GetThreadRemote(ctx context.Context, arg chat1.GetThrea
 			if arg.Query.Before != nil && msg.ServerHeader.Ctime > *arg.Query.Before {
 				continue
 			}
-			if mts != nil && !mts[msg.ServerHeader.MessageType] {
-				continue
-			}
+
 		}
 		res.Thread.Messages = append(res.Thread.Messages, *msg)
+		if mts != nil && mts[msg.ServerHeader.MessageType] {
+			count++
+		} else if mts == nil {
+			count++
+		}
```

Comment:
Super nit-pick, and doesn't really matter: `if mts == nil || mts[msg.ServerHeader.MessageType] {`